### ISSUE TITLE
Integrate with radare2

### ISSUE THREAD

**Issue by XVilka** [merrychap/shellen#20]
Would be awesome to make it work on top of radare2, since it has support for much more disassemblers and assemblers than capstone and keystone, getting a ton of useful features along with them.

**Comment by merrychap** [comment]
Yeah, I will think about this addition, but not sure that I will have time to implement it.
If you don't mind, I leave this issue open as a sign that it might be implemented in the future
